dim_checks/moz-fx-data-shared-prod/telemetry/active_users_aggregates/dim_checks.yaml

```diff
@@ -0,0 +1,103 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  partition_field: submission_date
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - submission_date
+          - client_id
+          - first_seen_date
+          - first_seen_year
+          - app_name
+          - app_version
+          - channel
+          - segment
+          - attributed
+          - os
+          - os_version
+          - os_version_major
+          - os_version_minor
+          - dau
+          - wau
+          - mau
+          - new_profiles
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: matches_regex
+      params:
+        columns:
+          - country
+        regex: ^[A-Z]{2}$  # example: GB
+    - type: value_in_set
+      params:
+        columns:
+          - channel
+        expected_values:
+          - nightly
+          - aurora
+          - release
+          - Other
+          - beta
+          - esr
+    - type: value_in_set
+      params:
+        columns:
+          - segment
+        expected_values:
+          - casual_user
+          - regular_user
+          - infrequent_user
+          - other
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - app_name
+        expected_values:
+          - "Firefox iOS"
+          - "Firefox iOS BrowserStack"
+          - "Firefox Desktop"
+          - "Fenix BrowserStack"
+          - Fenix
+          - "Focus iOS"
+          - "Focus Android"
+    - type: value_in_set
+      params:
+        columns:
+          - os
+        expected_values:
+          - iOS
+          - Android
+          - Linux
+          - Linux_CXL
+          - Darwin
+          - Windows_95
+          - Windows_98
+          - Windows_NT
+          - OpenBSD
+          - SunOS
+          - HubertOS
+    - type: numeric_value_matches
+      params:
+        columns:
+          - first_seen_year: "> 2020"
```

Comment:
@lucia-vargas-a is this a correct assumption? It seems we have a lot of values which would be less that that.